### PR TITLE
[ChangeLog] Fetch JSON from updated github location

### LINKED ISSUE #1521: Updates/Notes - Fetch JSON from correct location

### PR DESCRIPTION
**Pull request by meissadia**
Closes #1521

### CODE REVIEW

File: src/homepage/ForDataUsers/DataBrowser.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const DataBrowser = () => (
   <ExpandableCard
     id="home-expand-data-browser"
     title="HMDA Data Browser"
-    description="A suite of tools that allows financial institutions to filter, summarize, download, and visualize HMDA datasets."
+    description="A suite of tools that allows users to filter, summarize, download, and visualize HMDA datasets."
```
Comment (Michaeldremy) [inline src/homepage/ForDataUsers/DataBrowser.jsx:12]: Was there further discussion to update this text? The text to be changed was an update to the original via feedback from the powerpoint.

Comment (meissadia) [reply src/homepage/ForDataUsers/DataBrowser.jsx:12]: This was an update from @kgudel in https://github.com/cfpb/hmda-frontend/issues/1510.  Makes it clear that the tools aren't exclusively for FIs.

Comment (Michaeldremy) [reply src/homepage/ForDataUsers/DataBrowser.jsx:12]: I understand, thanks for the update. Looks good!

**Comment by Michaeldremy** [comment]
LGTM

**Pull request merged**